windows/make.vbs

```diff
@@ -1,2 +1,99 @@
+
+'Build script for KOLIBRI INSTALLER WINDOWS
+'
+'Steps
+'
+'1. Set Kolibri Version
+'2. Download python installer
+'3. Build the installation package
+'
+'Python Variable
+'PYTHON_VERSION == must set a specific verion (e.g. "2.7", "3.4.3" and etc.)
+'PYTHON_32BIT_LINK & PYTHON_64BIT_LINK == URL of the python .exe 
+'PYTHON_DIR == Directory when python must save
+'
+'REF: VB Scripting References
+'https://www.codeproject.com/tips/506439/downloading-files-with-vbscript
+'http://www.csidata.com/custserv/onlinehelp/vbsdocs/vbs14.htm
+'https://stackoverflow.com/questions/20626863/can-you-get-the-pc-user-name-in-vbs
+'https://blogs.technet.microsoft.com/heyscriptingguy/2005/02/08/how-can-i-find-and-replace-text-in-a-text-file/
+
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Function Downloads(strLink,path)
+'strLink == Get file name from URL.
+'path == for directory file
+strSaveName = Mid(strLink, InStrRev(strLink,"/") + 1, Len(strLink))
+strSaveTo = path & strSaveName
+' Create an HTTP object
+Set objHTTP = CreateObject("MSXML2.XMLHTTP")
+' Download the specified URL
+objHTTP.open "GET", strLink, False
+objHTTP.send
+ 
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+If objFSO.FileExists(strSaveTo) Then
+  objFSO.DeleteFile(strSaveTo)
+End If
+
+If objHTTP.Status = 200 Then
+  Dim objStream
+  Set objStream = CreateObject("ADODB.Stream")
+  With objStream
+    .Type = 1 'adTypeBinary
+    .Open
+    .Write objHTTP.responseBody
+    .SaveToFile strSaveTo
+    .Close
+  End With
+  set objStream = Nothing
+End If
+
+end Function
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+'Set Kolibri Version
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+set folder = objFSO.GetFolder ( left(WScript.ScriptFullName,(Len(WScript.ScriptFullName))-(len(WScript.ScriptName))) )
+for each file in folder.Files
+if instr (file.Name, ".whl") then
+whl = file.Name
+end if
+next

```

Comment:
I know this isn't Python, but the above block could really use some indentation :)

Reply:
Ok, @benjaoming I will check my code and put some indentation.